### DISCUSSION TITLE
Suggestion add SUI Dashboard

### DISCUSSION THREAD

**Discussion by kszere** [tteck/Proxmox#424] [Ideas]
[https://github.com/jeroenpardon/sui](url)

**Comment by MrEddX** [comment]
> [https://github.com/jeroenpardon/sui](url)

I'm using a very close version to this "SUI Dashboard", but from a different developer, who is also from Poland. It's called [**Flame Dashboard**](https://github.com/pawelmalak/flame). I don't know which one is the original, but I can say that it is extremely convenient and installs in no time.

By the way @kszere the above link does not work.
